scripts/gen_help_html.lua

```diff
@@ -44,6 +44,14 @@ local spell_dict = {
   viml = 'Vimscript',
   ['tree-sitter'] = 'treesitter',
   ['Tree-sitter'] = 'Treesitter',
+  colour = 'color',
```

Comment:
TODO: Add a title-case invariant check/regular word dict to avoid duplicating every word